db/migrate/20180128162224_sync_user_status_with_drupal_user_status.rb

```diff
@@ -0,0 +1,12 @@
+class SyncUserStatusWithDrupalUserStatus < ActiveRecord::Migration
+  def up
+	  DrupalUser.all.each do |du|	
```

Comment:
Hi! This looks awesome, thanks! Could you use 2-space indents here, though? Sorry to be so formatting inflexible :-)

Reply:
Yeah, I noticed that. Don't know why but I had 2 space indent in my text Editor and seems to distort here. No worries, I'll try and correct it!